assessments/27-28/day-01/readme.md

```diff
@@ -0,0 +1,173 @@
+# <img src="https://cloud.githubusercontent.com/assets/7833470/10423298/ea833a68-7079-11e5-84f8-0a925ab96893.png" width="60"> Day 1 Assessment
+
+**Instructions:** Do your best! Show your work - we’re more interested in your thought process and problem-solving skills than we are in exact syntax. Feel free to use code or "psuedo-code" (i.e. plain english).
+
+1. Fill in the blank below. When executed, what will the last line of code evaluate to?
+
+	```js
+	var student_name = __________;
+	var greeting = "Welcome to General Assembly, ";
+	console.log(greeting + student_name + "!");
+	```
+	<br>
+	<br>
+	<br>
+	<br>
+	<br>
+
+2. Translate the following webpage into HTML (do not use any CSS styling).
+
+	![simple-html-layout](https://cloud.githubusercontent.com/assets/1489337/13724295/031c7bd2-e836-11e5-882e-c2b24e9c6c9b.png)
+
+	<br>
+	<br>
+	<br>
+	<br>
+	<br>
+	<br>
+	<br>
+	<br>
+	<br>
+	<br>
+
+  **2.1** The HTML you just wrote can be described as having a “tree” structure. Draw a diagram which visualizes your HTML’s structure.
+
+	<br>
+	<br>
+	<br>
+	<br>
+	<br>
+	<br>
+	<br>
+	<br>
+	<br>
+	<br>
+
+3. Given the below code, how would you get the value `30`?
+
+	```js
+	var ages = [26, 28, 30, 28, 17];
+	```
+	<br>
+	<br>
+	<br>
+	<br>
+	<br>
+
+4. Given the below code, how would you add the name "Michael Nesmith" to the monkees?
+
+	```js
+	var monkees = ["Peter Tork", "Micky Dolenz", "Davy Jones"];
+	```
+	<br>
+	<br>
+	<br>
+	<br>
+	<br>
+
+	**4.1** How would you find the total number of band members?
+
+	<br>
+	<br>
+	<br>
+	<br>
+	<br>
+
```

Comment:
Let's add a question on how to get data out of an object, just to check what they remember from fundamentals:

Given the code below, how would you get the `street` part of the address?
	
	```js
	var address = {
		city: "San Francisco",
		number: 225,
		street: "Bush St.",
		state: "CA",
		zip: 94104
	};
	```
	<br>
	<br>
	<br>
	<br>
	<br>